velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -13,67 +13,336 @@
 #include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/functions/prestosql/registration/RegistrationFunctions.h"
+#include "velox/parse/TypeResolver.h"
+#include "velox/vector/tests/VectorTestBase.h"
 
 using namespace facebook::velox;
-using namespace facebook::velox::exec::test;
-class PlanNodeToStringTest : public OperatorTestBase {
+using facebook::velox::exec::test::PlanBuilder;
+
+class PlanNodeToStringTest : public testing::Test, public test::VectorTestBase {
  public:
   PlanNodeToStringTest() {
-    auto rowType = ROW({"c0", "c1"}, {INTEGER(), INTEGER()});
-
-    auto vector = makeRowVector(
-        {makeFlatVector<int32_t>(100, [](auto row) { return row; }),
-         makeFlatVector<int32_t>(100, [](auto row) { return row + 1; })});
-    vectors_.push_back(vector);
+    functions::prestosql::registerAllScalarFunctions();
+    parse::registerTypeResolver();
+    data_ = makeRowVector({
+        makeFlatVector<int16_t>({0, 1, 2, 3, 4}),
+        makeFlatVector<int32_t>({0, 1, 2, 3, 4}),
+        makeFlatVector<int64_t>({0, 1, 2, 3, 4}),
+    });
 
-    plan_ = exec::test::PlanBuilder()
-                .values(vectors_)
+    plan_ = PlanBuilder()
+                .values({data_})
                 .filter("c0 % 10 < 9")
                 .project({"c0 AS out1", "c0 % 100 + c1 % 50 AS out2"})
                 .filter("out1 % 10 < 8")
                 .project({"out1 + 10 AS out3"})
                 .planNode();
   }
 
+  RowVectorPtr data_;
   std::shared_ptr<core::PlanNode> plan_;
-
- private:
-  std::vector<RowVectorPtr> vectors_;
 };
 
 TEST_F(PlanNodeToStringTest, basic) {
-  auto output = plan_->toString();
-  ASSERT_EQ("->project\n", output);
+  ASSERT_EQ("->project\n", plan_->toString());
 }
 
 TEST_F(PlanNodeToStringTest, recursive) {
-  auto output = plan_->toString(false, true);
-  auto* expectedOutput =
+  ASSERT_EQ(
       "->project\n"
       "  ->filter\n"
       "    ->project\n"
       "      ->filter\n"
-      "        ->values\n"
-      "";
-  ASSERT_EQ(std::string(expectedOutput), output);
+      "        ->values\n",
+      plan_->toString(false, true));
 }
 
 TEST_F(PlanNodeToStringTest, detailed) {
-  auto output = plan_->toString(true, false);
   ASSERT_EQ(
       "->project[expressions: (out3:BIGINT, plus(cast ROW[\"out1\"] as BIGINT,10)), ]\n",
-      output);
+      plan_->toString(true, false));
 }
 
 TEST_F(PlanNodeToStringTest, recursiveAndDetailed) {
-  auto output = plan_->toString(true, true);
   ASSERT_EQ(
       "->project[expressions: (out3:BIGINT, plus(cast ROW[\"out1\"] as BIGINT,10)), ]\n"
       "  ->filter[expression: lt(mod(cast ROW[\"out1\"] as BIGINT,10),8)]\n"
-      "    ->project[expressions: (out1:INTEGER, ROW[\"c0\"]), (out2:BIGINT, plus(mod(cast ROW[\"c0\"] as BIGINT,100),mod(cast ROW[\"c1\"] as BIGINT,50))), ]\n"
+      "    ->project[expressions: (out1:SMALLINT, ROW[\"c0\"]), (out2:BIGINT, plus(mod(cast ROW[\"c0\"] as BIGINT,100),mod(cast ROW[\"c1\"] as BIGINT,50))), ]\n"
```

Comment:
missing spaces after the separator `,`. Not related to this change, but something to fix later.